docs/api-reference/operator.istio.io.md

```diff
@@ -0,0 +1,3242 @@
+| --- | --- | --- | --- |
+| `secretName` _string_ | Name of the secret the certificate and its key will be stored into.<br />If it is empty, it will not be stored into a secret.<br />Instead, the certificate and its key will be stored into a hard-coded directory. |  |  |
+| `dnsNames` _string array_ | The DNS names for the certificate. A certificate may contain<br />multiple DNS names. |  |  |
+
+
+#### ClientTLSSettings
+
+_Underlying type:_ _[struct{Mode ClientTLSSettingsTLSmode "json:\"mode,omitempty\""; ClientCertificate string "json:\"clientCertificate,omitempty\""; PrivateKey string "json:\"privateKey,omitempty\""; CaCertificates string "json:\"caCertificates,omitempty\""; CredentialName string "json:\"credentialName,omitempty\""; SubjectAltNames []string "json:\"subjectAltNames,omitempty\""; Sni string "json:\"sni,omitempty\""; InsecureSkipVerify *bool "json:\"insecureSkipVerify,omitempty\""; CaCrl string "json:\"caCrl,omitempty\""}](#struct{mode-clienttlssettingstlsmode-"json:\"mode,omitempty\"";-clientcertificate-string-"json:\"clientcertificate,omitempty\"";-privatekey-string-"json:\"privatekey,omitempty\"";-cacertificates-string-"json:\"cacertificates,omitempty\"";-credentialname-string-"json:\"credentialname,omitempty\"";-subjectaltnames-[]string-"json:\"subjectaltnames,omitempty\"";-sni-string-"json:\"sni,omitempty\"";-insecureskipverify-*bool-"json:\"insecureskipverify,omitempty\"";-cacrl-string-"json:\"cacrl,omitempty\""})_
+
+SSL/TLS related settings for upstream connections. See Envoy's [TLS
+context](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto.html#common-tls-configuration)
+for more details. These settings are common to both HTTP and TCP upstreams.
+
+
+For example, the following rule configures a client to use mutual TLS
+for connections to upstream database cluster.
+
+
+{{<tabset category-name="example">}}
+{{<tab name="v1alpha3" category-value="v1alpha3">}}
+```yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+
+
+	name: db-mtls
+
+
+spec:
+
+
+	host: mydbserver.prod.svc.cluster.local
+	trafficPolicy:
+	  tls:
+	    mode: MUTUAL
+	    clientCertificate: /etc/certs/myclientcert.pem
+	    privateKey: /etc/certs/client_private_key.pem
+	    caCertificates: /etc/certs/rootcacerts.pem
+
+
+```
+{{</tab>}}
+
+
+{{<tab name="v1beta1" category-value="v1beta1">}}
+```yaml
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+
+
+	name: db-mtls
+
+
+spec:
+
+
+	host: mydbserver.prod.svc.cluster.local
+	trafficPolicy:
+	  tls:
+	    mode: MUTUAL
+	    clientCertificate: /etc/certs/myclientcert.pem
+	    privateKey: /etc/certs/client_private_key.pem
+	    caCertificates: /etc/certs/rootcacerts.pem
+
+
+```
+{{</tab>}}
+{{</tabset>}}
+
+
+The following rule configures a client to use TLS when talking to a
+foreign service whose domain matches *.foo.com.
+
+
+{{<tabset category-name="example">}}
```

Comment:
This brackets are shown in the page as brackets alone

Reply:
Do you know from where this information comes? Do we create this example or does it also come from some upstream file? Those example has a lot of errors when they are shown in md format

Reply:
This `<tabset>` tag seems to be an upstream Istio documentation thing. Not sure what we can do about it. We can't change it in the upstream files, so maybe we need to transform them using our API transform tool.